go/common/crypto/signature/signature.go

```diff
@@ -666,3 +666,12 @@ func BuildPublicKeyBlacklist(allowTestKeys bool) {
 		blacklistedPublicKeys.Store(pk, true)
 	}
 }
+
+// NewBlacklistedKey returns the PublicKey from the given hex or panics.
+func NewBlacklistedKey(hex string) PublicKey {
```

Comment:
Based on the name this should actually blacklist the key.

Reply:
Thanks, fixed!